ext/pcntl/tests/pcntl_cpuaffinity.phpt

```diff
@@ -5,6 +5,7 @@ pcntl
 --SKIPIF--
 <?php
 if (!function_exists("pcntl_setcpuaffinity")) die("skip pcntl_setcpuaffinity is not available");
+<?php if (getenv('TRAVIS')) die('skip Currently fails on Travis'); ?>
```

Comment:
Confusingly, the end tag of the skipif was not closed, leading to this mistake that there is now a php start tag inside php code.

```suggestion
if (getenv('TRAVIS')) die('skip Currently fails on Travis');
?>
```

Reply:
hehe I admit wrong copy paste move :-)